SteinsKit.playground/Contents.swift

```diff
@@ -3,43 +3,76 @@ import UIKit
 import PlaygroundSupport
 import SteinsKit
 
+// MARK: - Model
+final class Model {
+    
+    private var _count: Variable<Int> = .init(0)
+    
+    private func _countUp() {
+        _count.accept({ $0 + 1 })
+    }
+    
+}
+
+extension Model: ModelProtocol {
+    
+    var count: AnyObservable<Int> {
+        return _count.asAnyObservable()
+    }
+    
+    func countUp() {
+        _countUp()
+    }
+    
+}
+// MARK: Model End -
+
+// MARK: - View Controller
 protocol ModelProtocol: AnyObject {
-    var title: AnyObservable<String> { get }
+    var count: AnyObservable<Int> { get }
+    func countUp()
 }
-final class MyViewController : UIViewController {
-    var model: ModelProtocol!
-    override func loadView() {
-        let view = UIView()
-        view.backgroundColor = .white
+
+final class ViewController : UIViewController {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/el-hoshino/SteinsKit/blob/14963d3a1945aa0404ae35196ade337826ee0ed4/SteinsKit.playground/Contents.swift#L36"></span>Colons should be next to the identifier when specifying a type and next to the key in dictionary literals.<br />
<code>colon</code> <code>Contents.swift:36</code></td>
    </tr>
  </tbody>
</table>